aws-s3/spring-cloud-starter-stream-source-s3/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -0,0 +1,4 @@
+configuration-properties.classes=org.springframework.cloud.stream.app.s3.source.AmazonS3SourceProperties, \
+  org.springframework.cloud.stream.app.file.FileConsumerProperties, \
+  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultUnlimited
```

Comment:
What is the point of this file if we still use `@EnableConfigurationProperties` ?
Someone externally is going to read it?
Why can't we parse `@EnableConfigurationProperties` on the `@EnableBinding` class instead?